spec/parsers/bulkrax/bagit_parser_spec.rb

```diff
@@ -452,51 +347,6 @@ module Bulkrax
           end
         end
       end
-
-      describe '#export_headers' do
```

Comment:
this method is now inherited from the csv parser and already being accounted for in the .csv parser spec.